<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="subtractor">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #00FFFF;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="adder">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #00FFFF;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Clear">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="MEMget">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #00FFFF;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="divider">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #00FFFF;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="Equals">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #00FFFF;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="MEMclear">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #00FFFF;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background-color: gray;
border: 1px solid gray;
color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="multiplier">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #00FFFF;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="ChangeSign">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>CS +/-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="MEMadd">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #00FFFF;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
